### DISCUSSION TITLE
Prompt for password change and MFA setup for new SOC user account

### DISCUSSION THREAD

**Discussion by Rbartram** [Security-Onion-Solutions/securityonion#9482] [General] [answered]
Hello, 

Is there a way to prompt SOC users to change their password on first login?  Also, is there a way to require MFA be setup on first login?

Thank you, ...Rob

**Comment by jertel** [comment, marked as answer]
Not currently. This has been on our internal todo list for a while now. I have added it to the issues list (#9484) for public visibility. Thanks.

In the meantime, SOC admins can (and should) review the Admin -> Users list periodically, to look for any users that do not have MFA enabled.

**Reply by Rbartram** [reply]
Thank you very much!
